plugins/generator-1.18.2/forge-1.18.2/procedures/entity_islooking_at_entity.java.ftl

```diff
@@ -0,0 +1,14 @@
+(new Object() {
+	public boolean isLooking(Entity _source, Entity _target, double area, boolean ignoreBlocks) {
+		Vec3 vec31 = new Vec3(_target.getX() - _source.getX(), (_target.getEyeY() - _target.getEyeHeight() * 0.5) - _source.getEyeY(),
+				_target.getZ() - _source.getZ());
+		double d0 = vec31.length();
+		double d1 = _source.getViewVector(1.0F).normalize().dot(vec31.normalize());
+		if (_source instanceof LivingEntity _liv) {
+			if (ignoreBlocks)

```

Comment:
I thought of something. As you use ignoreBlocks only for this condition, you could already use the FTL code here, instead of providing the value to the object and then, using this parameter.

Reply:
This did not work; will revert tomorrow